src/web/handlers/FlingGestureHandler.ts

```diff
@@ -5,7 +5,7 @@ import { AdaptedEvent, Config } from '../interfaces';
 import GestureHandler from './GestureHandler';
 
 const DEFAULT_MAX_DURATION_MS = 800;
-const DEFAULT_MIN_ACCEPTABLE_DELTA = 160;
+const DEFAULT_MIN_ACCEPTABLE_DELTA = 32;
```

Comment:
Even though it should be fine (and also this functionality can be implemented using `Pan`), I think it would be better if users could change this value. What do you think @j-piasecki?

Reply:
Yeah, having it set to 160 is a bit much. We could make it also use velocity to determine whether to activate (if the distance was shorter than the minimum delta, but the gesture was fast). Anyway, that's something for another PR, if we want to do this at all.